app/Jobs/CloneImagesOrVideos.php

```diff
@@ -136,7 +143,7 @@ public function handle()
                 }
             }
             if ($copy->files()->exists()) {
-                ProcessNewVolumeFiles::dispatch($copy);
+                (new PostProcessingVolumeCloning($copy))->handle();
```

Comment:
Why create a separate job class and then call it (synchronously) here? I suggest that you just add a new method to this job (e.g. `$this->postProcessCloning($copy)`) which includes all the code of `PostProcessingVolumeCloning::handle`.